src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/response/TaskHistoryResponse.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.controllers.response;
+
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.hmcts.reform.wataskmanagementapi.cft.entities.TaskHistoryResource;
+
+import java.util.List;
+
+@Builder(toBuilder = true)
+@Data
```

Comment:
As above we don't need to use this any more with `Record`s in Java 17.  Don't need to fix here if easier.